test/integration/cpm-test/CMakeLists.txt

```diff
@@ -4,17 +4,18 @@
 ##  SPDX-License-Identifier: MIT
 ##==================================================================================================
 cmake_minimum_required(VERSION 3.19)
```

Comment:
should we put the same requirements as the main CMakeLists.txt?

Reply:
Done.

Reply:
@jfalcou It looks like the CMake version on some of the CI environments is 3.18.4, which are now failing as a result of this update.

I can look into what version is absolutely mandatory. Maybe we can get away with 3.18.4 - maybe not.
 We could also just upgrade the CMake version. Thoughts?

Reply:
I was more thinking setting up everybody to 3.18 as we established I the other PR

Reply:
Oh, I understand what you were indicating. Will do.